frontend/src/views/fixed-nav-sticky-footer-view.js

```diff
@@ -0,0 +1,87 @@
+import {html, PolymerElement} from '@polymer/polymer/polymer-element.js';
+import '../components/ex-card.js';
+
+/**
+ * `fixed-nav-sticky-footer-view`
+ *
+ * FixedNavStickyFooterView element.
+ *
+ * @customElement
+ * @polymer
+ */
+class FixedNavStickyFooterView extends PolymerElement {
+
+    static get template() {
+        return html`
+            <style include="common-styles">
+                :host {
+                    display: flex;
+                    flex-direction: column;
+                    height: 100%;
+                }
+                
+                .wrapper {
+                    margin: 0 auto; /* Horizontally center the layout grid if there is space around it */
+                    max-width: 960px;
+                    padding: 0 var(--lumo-space-m);
+                }
+                
+                #header {
+                    z-index: 1;
+                    position: fixed;
+                    right: 0;
+                    left: 0;
+                    display: flex;
+                    white-space: nowrap;
+                    flex-wrap: wrap;
+                    align-items: center;
+                    padding: 0 var(--lumo-space-s);
+                    --lumo-primary-text-color: var(--lumo-contrast-90pct);
+                }
+                
+                .searchField {
+                    padding: var(--lumo-space-s) 0;
+                    margin-right: var(--lumo-space-s);
+                }
+                
+                #header a,
+                #header span {
+                    display: inline-block;
+                    margin: var(--lumo-space-s);
+                }
+                
+                #header .titleLink {
+                    font-size: var(--lumo-font-size-xl);
+                    margin-right: var(--lumo-space-m);
+                }
+                
+                main {
+                    flex-shrink: 0;
+                }
+                
+                #main {
+                    margin-top: 100px;
```

Comment:
IMO the margin should be on the `main` element (parent), not the child. 
We probably don't want to showcase how `margin` works in CSS.

Also, I would recommend to align `margin-top` with the header height.

Reply:
If the margin is moved to the `<main>` element, it doesn't collapse with the `<h1>` margin-top.

Note that the header gets bigger on smaller viewport when the contents wrap. I wanted to keep it simple but looking ok in small and big viewports. There's more margin on big viewport atm. and almost none when the viewport is around 324px wide. But I think it's fine as is.

Best would be to have the margin automatically calculated based on header height, but that would require some JS (and e.g. set the header height as a custom property). But I'd like to keep this simple. Maybe at some point we could add a component that does that header height calculation if it was needed for fixed header of multiple examples.

Reply:
Thanks for the clarification. Now I get the point, so it makes sense to keep it.
If we get some feedback regarding the code, we could try to revamp it later.